src/Controller/EmbedController.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller;
+
+use App\Repository\EmbedRepository;
+use App\Utils\Embed;
+use Psr\Log\LoggerInterface;
+use Symfony\UX\TwigComponent\Attribute\AsTwigComponent;
+use Symfony\UX\TwigComponent\Attribute\PostMount;
+
+#[AsTwigComponent('embed', template: 'components/embed.html.twig')]
+class EmbedController extends AbstractController
```

Comment:
this is a twig component though? it should go in `App\Twig\Components` (`src/Twig/Components/EmbedComponent.php`) and unless you somehow also need this to act as a controller it shouldn't extend the `AbstractController` and be named appropiately

```suggestion
namespace App\Twig\Components;

use App\Repository\EmbedRepository;
use App\Utils\Embed;
use Psr\Log\LoggerInterface;
use Symfony\UX\TwigComponent\Attribute\AsTwigComponent;
use Symfony\UX\TwigComponent\Attribute\PostMount;

#[AsTwigComponent('embed', template: 'components/embed.html.twig')]
class EmbedComponent
```

Reply:
You're absolutely right :+1: